uai2017/webfield/spc_group_webfield.html

```diff
@@ -0,0 +1,336 @@
+      } else {
+        combinedObj[reviewerNum] = _.assign({}, reviewerIds[reviewerNum]);
+      }
+    }
+    var reviewProgressData = {
+      numSubmittedReviews: Object.keys(completedReviews).length,
+      numReviewers: Object.keys(reviewerIds).length,
+      reviewers: combinedObj,
+      averageRating: ratings.length ? _.round(_.sum(ratings) / ratings.length, 2) : 'N/A'
+    };
+    var reviewHtml = Handlebars.templates.noteReviewers(reviewProgressData);
+
+    // Build Status Cell
+    var invitationUrlParams = {
+      id: note.forum,
+      noteId: note.id,
+      invitationId: CONFERENCE + '/-/Paper' + note.number + '/Meta/Review'
+    };
+    var reviewStatus = {
+      invitationUrl: '/forum?' + $.param(invitationUrlParams),
+      ratingUrl: '/invitation?id=' + CONFERENCE + '/-/Paper' + note.number + '/Area_Chair/Review/Rating',
+      ratingMax: 5
+    };
+    if (metaReview) {
+      reviewStatus.recommendation = metaReview.recommendation;
+      reviewStatus.editUrl = '/forum?id=' + note.forum + '&noteId=' + metaReview.id;
+    }
+    var statusHtml = Handlebars.templates.noteReviewStatus(reviewStatus);
+
+    return [number, summaryHtml, reviewHtml, statusHtml];
+  };
+
+  var buildNoteMap = function(noteNumbers) {
+    var noteMap = Object.create(null);
+    for (var i = 0; i < noteNumbers.length; i++) {
+      noteMap[noteNumbers[i]] = Object.create(null);
+    }
+    return noteMap;
+  };
+
+
+  // Kick the whole thing off
+  var headerLoaded = $.Deferred();
+  displayHeader(headerLoaded);
+
+  $.ajaxSetup({
+    contentType: 'application/json; charset=utf-8'
+  });
+
+  controller.addHandler('areachairs', {
+    token: function(token) {
+      var pl = model.tokenPayload(token);
+      var user = pl.user;
+
+      var userAreachairGroupsP = $.getJSON('groups', { member: user.id, regex: CONFERENCE + '/Paper.*/Area_Chair' })
+        .then(function(result) {
+          var noteNumbers = getPaperNumbersfromGroups(result.groups);
+          return $.when(
+            getBlindedNotes(noteNumbers),
+            getOfficialReviews(noteNumbers),
+            getMetaReviews(),
+            getReviewerGroups(noteNumbers),
+            getTagInvitations(),
+            headerLoaded
+          );
+        })
+        .then(function(blindedNotes, officialReviews, metaReviews, noteToReviewerIds, tagInvitations, loaded) {
+          displayStatusTable(blindedNotes, officialReviews, metaReviews, noteToReviewerIds, '#reviewer-status');
+        })
+        .fail(function(error) {
+          displayError();
+        });
+    }
+  });
+
+  $('#group-container').on('click', 'a.note-contents-toggle', function(e) {
+    var hiddenText = 'Show paper details';
+    var visibleText = 'Hide paper details';
+    var updated = $(this).text() === hiddenText ? visibleText : hiddenText;
+    $(this).text(updated);
+  });
+
+  $('#group-container').on('click', 'a.send-reminder-link', function(e) {
+    var userId = $(this).data('userId');
```

Comment:
@zbialecki is it possible to access to the note id to send the link to the forum in the email content?

```
This is a reminder to please submit your official reviews for UAI 2017 to this paper: https://openreview.net/forum?id=[note]
```

or 

```
This is a reminder to please submit your official reviews for UAI 2017. Click on the following link to submit your review: https://openreview.net/forum?id=[note]&noteId=[note]&invitationId=[invitationId]
```